### PR TITLE
Support for Min aggregation

### PR DESCRIPTION
**Pull request by Mimetis**
The following changes are proposed:

* min aggregation

### CODE REVIEW

**Review by YannickBrombach — COMMENTED** [review]
Looking good to me

**Review by eladiw — APPROVED** [review]

File: K2Bridge/Visitors/MetricAggregationVisitor.cs [hunk 1]
```diff
@@ -31,6 +31,15 @@ public void Visit(CardinalityAggregation cardinalityAggregation)
             cardinalityAggregation.KustoQL = $"['{cardinalityAggregation.FieldAlias}']={KustoQLOperators.DCount}({cardinalityAggregation.FieldName})";
         }
 
+        /// <inheritdoc/>
+        public void Visit(MinAggregation minAggregation)
```
Comment (AsafMah) [inline K2Bridge/Visitors/MetricAggregationVisitor.cs:35]: Nit: in all places except here max comes before min. Personally I think min should become before max, but it's up to you, just stay consistent.

**Review by AsafMah — APPROVED** [review]
Looks good

File: K2Bridge/Visitors/MetricAggregationVisitor.cs (hunk 1 above)
Comment (Mimetis) [reply K2Bridge/Visitors/MetricAggregationVisitor.cs:35]: Indeed, make more sense. More readable, from a human perspective

**Pull request merged**